engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetStateParameter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.lwjgl.opengl.GL11;
+import org.terasology.rendering.dag.RenderPipelineTask;
+import org.terasology.rendering.dag.StateChange;
+
+/**
+ * TODO: Add javadocs
+ * Indented for capabilities that are enabled/disabled via glEnable and glDisable.
+ */
+public abstract class SetStateParameter implements StateChange {
+    private static Map<Integer, String> paramaterMap = ImmutableMap.of(GL11.GL_BLEND, "GL_BLEND",
+            GL11.GL_DEPTH_TEST, "GL_DEPTH_TEST",
+            GL11.GL_STENCIL_TEST, "GL_STENCIL_TEST",
+            GL11.GL_CULL_FACE, "GL_CULL_FACE");
```

Comment:
I've been thinking. What are we gaining in having this map here? It's a lot of "knowledge" aggregated into a class. Ideally that knowledge is placed close to where it's needed. And it's needed for the log output of tasks. 

So, how about:

`disablingTask = new DisableStateParameterTask(PARAMETER);`

becomes

`disablingTask = new DisableStateParameterTask(int parameter, String parameterName);`

This way we do not need a central repository for this information and individual state changes can provide the appropriate name to the disabling/enabling task.